aten/src/ATen/core/function_schema.h

```diff
@@ -146,12 +154,17 @@ struct FunctionSchema {
     return is_varret_;
   }
   bool is_mutable() const {
-    return std::any_of(
-        arguments_.cbegin(), arguments_.cend(), [](const Argument& arg) {
-          const auto& aliasInfo = arg.alias_info();
-          return aliasInfo && aliasInfo.value().isWrite();
-        });
+    return has_alias_annotation_ ||
```

Comment:
Nit: Is the intention here that unschematized user ops will return true even if they don't have any aliasing inputs?  If yes, add comment ? if no, don't need the `has_alias_annotation_` case